### PR TITLE
cmd/newapp: Wait longer for imagestream import to finish

### LINKED ISSUE #15015: test/cmd/newapp.sh test flake - late completion of repository import
```
=== BEGIN TEST CASE ===
test/cmd/newapp.sh:29: executing 'oc get imagestreamtags mysql:5.6' expecting success; re-trying every 0.2s until completion or 60.000s
FAILURE after 59.644s: test/cmd/newapp.sh:29: executing 'oc get imagestreamtags mysql:5.6' expecting success; re-trying every 0.2s until completion or 60.000s: the command timed out
Standard output from the command:
Standard error from the command:
Error from server (NotFound): imagestreamtags.image.openshift.io "mysql:5.6" not found
... repeated 106 times
=== END TEST CASE ===
```

In [this job ](https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_check/4086/testReport/junit/github/com_openshift_origin_test_cmd_newapp/test_cmd_newapp_sh_29__executing__oc_get_imagestreamtags_mysql_5_6__expecting_success__re_trying_every_0_2s_until_completion_or_60_000s/).

In the master log I can see that the import has been finished a while after the lastest check for the `mysql:5.6` istag:

```
I0703 11:27:33.987853   10348 wrap.go:42] GET /oapi/v1/namespaces/cmd-newapp/imagestreamtags/mysql:5.6: (2.131133ms) 404 [[oc/v1.6.1+5115d708d7 (linux/amd64) kubernetes/314edd5] 172.17.0.2:47492]
I0703 11:32:05.890608   10348 factory.go:185] Image stream "mysql" updated.
``` 

Late by 5 seconds. Shall we extend the timeout to 2 minutes?

### PR DESCRIPTION
**Pull request by miminar**
Resolves #15015

### CODE REVIEW

**Comment by miminar** [comment]
Really increased the timeout to 2 minutes instead of 120 milliseconds.

**Comment by dmage** [comment]
just want to check CI stuff
/test extended_image_registry
/test extended_image_ecosystem

**Comment by miminar** [comment]
@kargakis ping

File: test/cmd/newapp.sh [hunk 1]
```diff
@@ -24,10 +24,10 @@ os::cmd::expect_success 'oc new-app https://github.com/openshift/rails-ex --stra
 os::cmd::expect_success_and_text 'oc new-app mongo -o yaml' 'image:\s*mongo'
 # the local image repository takes precedence over the Docker Hub "mysql" image
 os::cmd::expect_success 'oc create -f examples/image-streams/image-streams-centos7.json'
-os::cmd::try_until_success 'oc get imagestreamtags mysql:latest'
-os::cmd::try_until_success 'oc get imagestreamtags mysql:5.5'
-os::cmd::try_until_success 'oc get imagestreamtags mysql:5.6'
-os::cmd::try_until_success 'oc get imagestreamtags mysql:5.7'
+os::cmd::try_until_success 'oc get imagestreamtags mysql:latest' $((2*60*1000))
```
Comment (bparees) [inline test/cmd/newapp.sh]: os::util::environment::setup_time_vars

and then

2*TIME_MIN

**Pull request merged**